src/components/ToDo/ToDoForm.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react'
+import { Rectangle, Circle, TextBox } from './styles/ToDoCardForm.styled'
+
+type todo = {
+  id: string
+  task: string
+  completed: boolean
+}
+
+const ToDoForm = ({ addTodo, isDarkMode }: any) => {
```

Comment:
are these of unknown types?

Reply:
what is the type of addTodo ah? should be a function

Reply:
update: changed alr!